lib/dish.rb

```diff
@@ -0,0 +1,9 @@
+class Dish
```

Comment:
Nice, simple, easy to read class. I'm not an expert, but maybe the methods could have been included on a different class? Although I'm sure you had a good reason why you chose this approach!